pkg/talos/helper.go

```diff
@@ -25,7 +25,7 @@ import (
 
 func ipDescovery(nodeIPs []string, ifaces []network.AddressStatusSpec) (publicIPv4s, publicIPv6s []string) {
 	for _, iface := range ifaces {
-		if iface.LinkName == "kubespan" || iface.LinkName == "lo" {
+		if iface.LinkName == "kubespan" || iface.LinkName == "lo" || strings.HasPrefix(iface.LinkName, "dummy") {
```

Comment:
probably also want to skip `siderolink` here?

Reply:
Yep, thanks!